src/types/ErrorTypes.ts

```diff
@@ -0,0 +1,9 @@
+export type ErrorType = 'load' | 'title' | 'add' | 'delete' | 'update';
+
+export enum Errors {
+  load = 'Unable to load todos',
+  title = 'Title should not be empty',
+  add = 'Unable to add a todo',
+  delete = 'Unable to delete a todo',
+  update = 'Unable to update a todo',
```

Comment:
Names in enum should be in upercase. 
There is no need for ErrorType when u have enum Errors, u can take it from there